### PR TITLE
Add first version of Buck build workflow

### PR DESCRIPTION
**Pull request by kit1980**

### CODE REVIEW

File: .github/workflows/buck_build_test.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+      - name: Install Buck
+        run: |
+          wget https://github.com/facebook/buck/releases/download/v2021.01.12.01/buck.2021.01.12.01_all.deb
+          sudo apt install ./buck.2021.01.12.01_all.deb
+
+      - name: Build Buck target
+        run: |
+          touch .buckconfig
+          buck build third_party:fmt
```
Comment (linbinyu) [inline .github/workflows/buck_build_test.yml:69]: use this command to build all targets in this folder (we will add more) 
buck build --keep-going //third_party:

**Review by linbinyu — CHANGES_REQUESTED** [review]

**Review by linbinyu — APPROVED** [review]

**Review by seemethere — APPROVED** [review]

**Pull request closed without merging**